cfgov/v1/tests/wagtail_pages/helpers.py

```diff
@@ -21,3 +38,336 @@ def publish_page(child):
+                    'language': True,
+                }
+            }
+        ])
+
+    try:
+        parent.add_child(instance=new_page)
+        new_page.save_revision().publish()
+    except ValidationError:
+        print("skipping page creation")
+
+    # return path
+    return new_page.get_url(None, site)
+
+def create_blog_page(page_title, page_slug, parent_path=None, \
+    page_tags={}, page_categories={}, page_language='en', \
+        date_published_override=date.today()):
+    # create a new page and set it as the child of an existing page
+    # return list of route paths
+    # get the root of the current site
+    site_model = apps.get_model('wagtailcore', 'Site')
+    site = site_model.objects.get(is_default_site=True)
+    root = site.root_page
+    # since parent was not provided, make root
+    parent = root
+
+    # check for optional variables set to None
+    if not page_language:
+        page_language='en'
+    if not page_tags:
+        page_tags={}
+    if not page_categories:
+        page_categories={}
+    if not date_published_override:
+        date_published_override=date.today()
+
+    # if a parent path is provided, use that as parent
+    if parent_path:
+        path_components = \
+            [component for component in parent_path.split('/') if component]
+
+        try:
+            route = root.route(None, path_components)
+        except Http404:
+            print("skipping page creation")
+
+        parent = route.page
+
+    # create page, add it as a child of parent, save, and publish
+    new_page = BlogPage(title=page_title, slug=page_slug, \
+        language=page_language, \
+            date_published=date_published_override)
+
+    # add tags and categories
+    for item in page_tags:
+        new_page.tags.add(item)
+    for item in page_categories:
+        new_page.categories.add(CFGOVPageCategory(name=item))
+
+    try:
+        parent.add_child(instance=new_page)
+        new_page.save_revision().publish()
+    except ValidationError:
+        print("skipping page creation")
+
+    # return path
+    return new_page.get_url(None, site)
+
+def create_browse_filterable_page(page_title, page_slug, parent_path=None, \
+    has_filter=True, filter_is_expanded=False):
+    # create a new page and set it as the child of an existing page
+    # return list of route paths
+    # get the root of the current site
+    site_model = apps.get_model('wagtailcore', 'Site')
+    site = site_model.objects.get(is_default_site=True)
+    root = site.root_page
+    # since parent was not provided, make root
+    parent = root
+    # if a parent path is provided, use that as parent
+    if parent_path:
+        path_components = \
+            [component for component in parent_path.split('/') if component]
+
+        try:
+            route = root.route(None, path_components)
+        except Http404:
+            print("skipping page creation")
+
+        parent = route.page
```

Comment:
Since this pattern is common across several of these functions, do you think it'd be worth breaking out into its own function? I could see this being generally useful to have for our unittests as well, where we have to fetch the default site and its root page.

Other thing I'm curious about — is there a reason you're using `apps.get_model()` here rather than `Site.objects.get(is_default_site=True)`?